src/streamlink/plugins/chzzk.py

```diff
@@ -174,19 +211,21 @@ def _get_live(self, channel_id):
                     ffmpeg_options={"copyts": True},
                 )
 
-    def _get_video(self, video_id):
-        datatype, data = self._api.get_videos(video_id)
+    def _get_vod_playback(self, datatype, data):
         if datatype == "error":
             log.error(data)
             return
 
-        self.id = video_id
-        in_key, vod_id, self.author, self.category, self.title, adult = data
+        adult, in_key, vod_id, *metadata = data
 
         if in_key is None or vod_id is None:
             log.error(f"This stream is for {'adults only' if adult else 'unavailable'}")
```

Comment:
Please also quickly fix the error message (here and the other one). "for" needs to be moved to the "adults only" string.